core/src/components/menu/test/safe-area/menu.e2e.ts

```diff
@@ -0,0 +1,117 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * This behavior does not vary across modes.
+ */
+configs({ modes: ['md'] }).forEach(({ title, config, screenshot }) => {
+  test.describe(title('menu: safe area'), () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`/src/components/menu/test/safe-area`, config);
+    });
+    test.describe('side: start', () => {
+      test('should render with safe area when notch is on the left', async ({ page }) => {
+        const ionDidOpen = await page.spyOnEvent('ionDidOpen');
+        const ionDidClose = await page.spyOnEvent('ionDidClose');
+
+        await page.evaluate(() => {
+          const style = document.createElement('style');
+          style.innerHTML = `
+            :root {
+              --ion-safe-area-left: 50px;
+              --ion-safe-area-right: 10px;
+            }
+          `;
+          document.head.appendChild(style);
```

Comment:
I know this new `<style>` will take priority since it's lower on the page, but it seems strange to append a new one when safe area styles already exist on the page we're visiting. Could we query for the existing style tag and update its innerHTML instead?